src/woo_publications/publications/migrations/0004_document_uuid_alter_document_identifier.py

```diff
@@ -15,6 +21,14 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.AddField(
+            model_name="document",
+            name="uuid",
+            field=models.UUIDField(
+                default=uuid.uuid4, editable=False, verbose_name="UUID"
+            ),
+        ),
+        migrations.RunPython(generate_unique_uuids, migrations.RunPython.noop),  # type: ignore reportArgumentType
```

Comment:
I think the type checker error here is a real error because you defined a method, not a simple function.